src/pages/token/rewards/LaunchIncentivesPanel.tsx

```diff
@@ -33,7 +33,7 @@ export const LaunchIncentivesPanel = ({ className }: { className?: string }) =>
 
   useEffect(() => {
     dispatch(markLaunchIncentivesSeen());
-  }, []);
+  }, [dispatch]);
```

Comment:
`exhaustive-hook-deps` 😭 

`dispatch` and `setStates` are never re-initialized so it isn't necessary to add as a dep, not sure how we can workaround this eslint rule

Reply:
Yeah setStates are handled correctly - never required as a dep but dispatch is a library so isn't built into eslint as an exception. I vote we just do as eslint demands. It can auto fix in vscode and it is simple.

Reply:
Googling a bit it seems that dispatch actually should get handled correctly and I have no idea why it's not.